package/cfg/config.yaml

```
@@ -262,28 +275,69 @@ target_mapping:
     - "etcd"
     - "policies"    
   "cis-1.24":
+    - "master"
+    - "node"
+    - "controlplane"
+    - "etcd"
+    - "policies"  
+  "cis-1.7":
     - "master"
     - "node"
     - "controlplane"
     - "etcd"
     - "policies" 
+
+# RKE1
+# rke1: Generic
   "rke-cis-1.4":
     - "master"
     - "node"
     - "etcd"
-  "rke-cis-1.5-hardened":
+# rke1: Permissive
+  "rke-cis-1.5-permissive":
     - "master"
     - "node"
     - "controlplane"
     - "etcd"
     - "policies"
-  "rke2-cis-1.5-hardened":
+    "rke-cis-1.6-permissive":
```

Could you fix the indentation so it's consistent?
```suggestion
  "rke-cis-1.6-permissive":
```